### PR TITLE
Add an Option to Filter Out PTR Zones in the Zones View

### PR DESCRIPTION
**Pull request by nspadaccino**
Fixes https://github.com/vinyldns/vinyldns/issues/1122

A checkbox option to filter out PTR zones has been added in the zones view. This filter will still apply when searching for zone names

### CODE REVIEW

File: modules/portal/app/views/zones/zones.scala.html [hunk 1]
```diff
@@ -50,6 +49,19 @@ <h3><span class="fa fa-table"></span> Zones</h3>
...
+<!--                                                <form action="">-->
+<!--                                                    <div class="checkbox pull-right">-->
+<!--                                                        <label>-->
+<!--                                                            <input type="checkbox" ng-model="includeReverse" ng-true-value="false" ng-false-value="true" ng-change="refreshZones()"> Hide PTR Zones-->
+<!--                                                        </label>-->
+<!--                                                    </div>-->
+<!--                                                </form>-->
+<!--                                            </div>-->
+<!--                                        </div>-->
+<!--                                    </div>-->
```
Comment (Aravindh-Raju) [inline modules/portal/app/views/zones/zones.scala.html:64]: We can remove this.

**Review by Aravindh-Raju — CHANGES_REQUESTED** [review]

**Review by Aravindh-Raju — CHANGES_REQUESTED** [review]
When both `Hide PTR Zones` and `Search by admin group` checkboxes are checked, the PTR Zones still shows up.

**Comment by nspadaccino** [comment]
Raised request on wrong repo

**Comment by arpit4ever** [comment]
New PR https://github.com/vinyldns/vinyldns/pull/1274

**Pull request closed without merging**